### PR TITLE
feat: 37_package.md.fr

### PR DESCRIPTION
**Pull request by Gmyard2019**

### CODE REVIEW

File: src/regions/ca/info/faq/37_package.md.fr [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Est t’il sécuritaire de reçevoir un colis?
+
+Oui, selon l’[Organisation Mondiale de la Santé](https://www.who.int/fr/emergencies/diseases/novel-coronavirus-2019/advice-for-public/q-a-coronaviruses), la chance que le paquet ait été contaminée par quelqu’un porteur du Covid est faible, ainsi que la chance que le virus est survécu au trajet.
+
+Cependant, si vous souhaitez éliminer à 100% tout risque voici [les instructions à suivre](https://www.niaid.nih.gov/news-events/new-coronavirus-stable-hours-surfaces?utm_source=share&utm_medium=ios_app&utm_name=iossmf (edited)):
```
Comment (hollyorlando) [inline src/regions/ca/info/faq/37_package.md.fr]: This link isn't formatted correctly :)

Comment (Gmyard2019) [reply src/regions/ca/info/faq/37_package.md.fr]: fixed!

Comment (hollyorlando) [reply src/regions/ca/info/faq/37_package.md.fr]: Doesn't look fixed on this PR yet :) @Gmyard2019

Comment (Gmyard2019) [reply src/regions/ca/info/faq/37_package.md.fr]: I put a new link but everything looked fine to me. Let me know if it does not work and if it does, could you point out exactly what is wrong?

**Pull request merged**